.pre-commit-config.yaml

```diff
@@ -28,3 +28,11 @@ repos:
         rev: v2.6.0
         hooks:
             -   id: reorder-python-imports
+
+    - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+      rev: v2.10.0
+      hooks:
+        - id: pretty-format-java
+          args: [--autofix, --aosp]
+        - id: pretty-format-kotlin
+          args: [--autofix]
```

Comment:
Ah, I was going to ask about autoformatting Java now all that dirty Python is gone, and there it is! Do you think it could be worth adding something for XML, as well?

Reply:
That would be nice. I was looking at https://pre-commit.com/hooks.html and the only thing that really popped out was https://github.com/comkieffer/xml-linter-hook. However, `xmllint --format` isn't that great and the hook actually doesn't support it. I'm all ears if you have any ideas!